pkg/server/endpoints/endpoints_test.go

```diff
@@ -240,7 +240,6 @@ func TestListenAndServe(t *testing.T) {
 	prepareDataStore(t, ds, []*testca.CA{ca, federatedCA}, agentSVID)
 
 	ctx, cancel := context.WithTimeout(context.Background(), time.Minute)
-	defer cancel()
```

Comment:
Was thi removed accidentally?

Reply:
no, the context is explicitly cancelled on ~L391 so this appeared duplicative.

Reply:
I think it's still better to keep. This makes sure it gets canceled  if the test is terminated early due to an assertion.

It also matches how you'd normally use the context. A defer to cancel it when exiting the function + optionally manually calling cancel early when that makes sesne, like here to stop the server.

Reply:
done!